kubernetes/overlays/lfa/enqueue-deploy.yaml

```diff
@@ -40,7 +40,7 @@ spec:
         - name: REDIS_HOST
           value: redis
         - name: DATASET_REGEXP
-          value: "(FiberSpectrograph:.*\\.fits$|GenericCamera/.*_\\d{6}\\.fits$)"
+          value: "(FiberSpectrograph:.*\\.fits$|GenericCamera/.*_\\d{6}\\.fits$|MTCamera/photodiode/.*\\.ecsv)"
```

Comment:
Having the extra `$` at the end provides a bit of safety (in case someone comes up with `.ecsv.gz` files, for example).